src/routes.tsx

```diff
@@ -12,6 +13,7 @@ const Routes: React.FunctionComponent = () => {
                 <Header />
                 <Route exact path="/" component={App} />
                 <Route exact path="/acquire" component={AcquirePage} />
+		        <Route exact path="/user" component={UserPage} />
```

Comment:
Odd indentation, did tabs sneak in?

Reply:
this has been fixed in https://github.com/bluesky/bluesky-webclient/pull/21/commits/d0cf07c7c71b3477f1002b3af0478a4da93ef309